bin/compile

```diff
@@ -23,7 +23,7 @@ APP_DEPS=$(cat ${ENV_DIR}/APP_DEPS)       # ex. `api core`
 
 (
     stage=`mktemp -d`                                                          && # make a temp directory to stage "correct" layout
-    mv ${BUILD_DIR}/${APP_BASE}/* ${stage}                                     && # stage the app folder we care about
+    mv ${BUILD_DIR}/${APP_BASE}/{,.[!.]}* ${stage}                             && # stage the app folder we care about
```

Comment:
This is quite terse. The brackets expand to make this:

```
mv ${BUILD_DIR}/${APP_BASE}/* ${BUILD_DIR}/${APP_BASE}/.[!.]* ${stage}
```

And `/.[!.]*` is `/.*` but not `/..*`